src/applications/check-in/pre-check-in/pages/Error/index.jsx

```diff
@@ -48,9 +48,9 @@ const Error = ({ location }) => {
     <>
       {messages.map((message, index) => {
         return (
-          <p key={index} data-testid={message.testId}>
+          <div key={index} data-testid={message.testId}>
```

Comment:
changed to a div to bring in line with other va-alerts in the app and help with alignment of the icon and the text (p has extra margin).